backend/workflow_manager/workflow_v2/models/file_history.py

```diff
@@ -39,11 +39,12 @@ def is_completed(self) -> bool:
         db_comment="Error message",
     )
     result = models.TextField(blank=True, db_comment="Result from execution")
+    meta_data = models.TextField(blank=True, db_comment="MetaData from execution")
```

Comment:
@muhammad-ali-e Suggest to rename field to `metadata`.

Reply:
Done